src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -80,6 +80,36 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
         return $filter & Caster::EXCLUDE_VERBOSE ? $z : $z + $a;
     }
 
+    public static function castPeriod(\DatePeriod $p, array $a, Stub $stub, $isNested, $filter)
+    {
+        if (defined('HHVM_VERSION_ID') || \PHP_VERSION_ID < 50605) {
+            return $a;
+        }
+
+        $dates = array();
+        if (\PHP_VERSION_ID >= 70107) { // see https://bugs.php.net/bug.php?id=74639
+            foreach (clone $p as $i => $d) {
+                $dates[] = sprintf('%s) %s', $i + 1, $d->format('Y-m-d H:i:s'));
+                if (11 === $i) {
+                    $dates[] = '...';
+                    break;
+                }
+            }
+        }
+
+        $period = sprintf(
+            'every %s, from %s (%s) %s',
+            self::formatInterval($p->getDateInterval()),
+            $p->getStartDate()->format('Y-m-d H:i:s'),
+            $p->include_start_date ? 'included' : 'excluded',
+            ($end = $p->getEndDate()) ? 'to '.$end->format('Y-m-d H:i:s') : 'recurring '.$p->recurrences.' time/s'
+        );
+
+        $p = array(Caster::PREFIX_VIRTUAL.'period' => new ConstStub($period, implode(PHP_EOL, $dates)));
```

Comment:
Purely out of interest, I too would like to know why the change. Is it simply a Symfony convention or is there an another underlying reason for such?